### DISCUSSION TITLE
Google Analitics

### DISCUSSION THREAD

**Discussion by fessnecro** [shlinkio/discussions#122] [Help wanted] [answered]
Is any way to use GA for short links?

**Comment by acelaya** [comment, marked as answer]
It [was considered](https://github.com/shlinkio/shlink/issues/263) time ago, and finally discarded for two reasons mainly:

* Complexity, due to limited support by Google to integrate using the technologies in Shlink's stack
* Closed data philosophy.

There are plans to [export visits stats to matomo](https://github.com/shlinkio/shlink/issues/1798) though, which goes more in line with Shlink in both points above.

**Comment by fessnecro** [comment]
Maybe it's good idea to add some layer with JS before redirect? Maybe i dont want to use GA or matomo. For example I wanna use yandex metrica.

**Comment by fessnecro** [comment]
I can use nginx for this mission. But I think it's bad idea)
